Making Sense Project API/Controllers/CarController.cs

```diff
@@ -1,82 +1,101 @@
-﻿using Making_Sense_Project_API.Model.Class;
+﻿using AutoMapper;
+using Making_Sense_Project_API.Model.Class;
+using Making_Sense_Project_API.Model.Dtos;
 using Making_Sense_Project_API.Model.Repository;
 using Microsoft.AspNetCore.Mvc;
 using System.Collections.Generic;
 using System.Linq;
 
 namespace Making_Sense_Project_API.Controllers
 {
-    [Route("api/[controller]")]
+    [Route("api/cars")]
```

Comment:
Por qué no mantener `[controller]` como en las otras controladoras? 
```suggestion
    [Route("api/[controller]")]
```